libs/langchain/langchain/cache.py

```diff
@@ -267,6 +283,8 @@ def update(self, prompt: str, llm_string: str, return_val: RETURN_VAL_TYPE) -> N
                 str(idx): generation.text for idx, generation in enumerate(return_val)
             },
         )
+        if self.ttl > 0:
+            self.redis.expire(key, self.ttl)
```

Comment:
Are we able to do this using a pipeline to avoid an extra network request for setting the ttl?